docs/Migrating.md

```diff
@@ -1,5 +1,17 @@
 # Migrating
 
+## Migrating from ML-Agents toolkit v0.4 to v0.5
+
+### Important
+ * We reorganized the entire repository
+ * `unity-environment` has been renamed `MLAgentsSDK`.
+ *  The `python` folder has been reorganized. We removed the `python` folder and replaced it with `ml-agents`. The folder `ml-agents` now contains the packages `mlagents.env` to interact directly with Unity environments and `mlagents.trainers` to train environments.
```

Comment:
Change too ....

The `python` folder has been renamed to `ml-agents`. It not contains two packages, `mlagents.env` and `mlagents.trainers`. `mlagents.env` can be used to interact directly with a Unity environment, while `mlagents.trainers` contains the classes for training agents.